include/z64message.h

```diff
@@ -292,8 +350,8 @@ typedef struct MessageContext {
     /* 0x120AC */ s16 stickXRepeatTimer;
     /* 0x120AE */ s16 stickYRepeatTimer;
     /* 0x120B0 */ u8 ocarinaSongEffectActive;
-    /* 0x120B1 */ u8 unk120B1;
-    /* 0x120B2 */ u8 unk120B2[0xA];
+    /* 0x120B1 */ u8 bombersNotebookNewEventQueueSize;
+    /* 0x120B2 */ u8 bombersNotebookNewEventQueue[0xA];
```

Comment:
Now that I think about it, it's more of a stack than a queue. Entries are added to the top of the stack, and then processed first-in-last-out. Probably worth renaming this to stack.